druid-shell/src/backend/windows/keyboard.rs

```diff
@@ -547,17 +550,17 @@ impl KeyboardState {
     /// is likely low, though.
     pub(crate) unsafe fn process_message(
         &mut self,
-        hwnd: HWND,
         msg: UINT,
         wparam: WPARAM,
         lparam: LPARAM,
+        is_last: bool,
```

Comment:
I'm wondering if "is_last_key" might be a better name, because at this place the context that it's keyboard specific is lost.